hotsos/core/plugins/kernel/net.py

```diff
@@ -394,3 +396,239 @@ def __getattr__(self, fld):
 
         raise AttributeError("{} not found in {}".
                              format(fld, self.__class__.__name__))
+
+
+class STOVParser(abc.ABC):
```

Comment:
I really like what this is doing and thing that it is some that could be generally useful. I am currently exploring how we could have FileSearcher perform these mappings natively so that e.g. when you go to retrieve your search results you can just retrieve them by (column) name rather than get(1) etc. I will report back asap on whether ive managed to get this to work and if it makes sense to just land this and adjust later we can do that too.

Reply:
searchkit now supports naming results fields but I'll go ahead and land this with the aim of proposing a switch to searchkit as a separate patch.